schemas/com.snowplowanalytics.snowplow.ecommerce/refund/jsonschema/1-0-0

```diff
@@ -0,0 +1,43 @@
+{
+    "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+    "description": "Schema for a refund in Ecommerce",
+    "self": {
+        "vendor": "com.snowplowanalytics.snowplow.ecommerce",
+        "name": "refund",
+        "format": "jsonschema",
+        "version": "1-0-0"
+    },
+    "type": "object",
+    "properties": {
+        "transaction_id": {
+            "description": "The ID of the transaction.",
+            "type": "string",
+            "maxLength": 4096
+        },
+        "currency": {
+            "description" : "The currency in which the product is being priced (ISO 4217).",
+            "type" : "string",
+            "maxLength": 3,
+            "minLength": 3
+        },
+        "refund_amount" : {
+            "type": "number",
+            "description": "The monetary amount refunded.",
+            "multipleOf": 0.01,
```

Comment:
Nit: There are currencies (the dinar) with three decimal places. I think we are safe to remove `multipleOf` here.

Reply:
Really good observation. AFAIK we use this in other places which has some efficiencies (a [past comment](https://github.com/snowplow/iglu-central/pull/1243/files#r1048228802)), but does that mean we cannot take advantage of multipleOf for monetary values ? _There are some other places that I saw this pattern used in the past._

I will remove it.

Reply:
We could maybe go to 3 decimal places but it is a fair bit of an edge case, and limiting the scale of a decimal is useful in terms of storage, and we do want to make sure that for currencies that schemaddl results in a decimal, rather than a float.

Reply:
@istreeter We need your input here on if it will play well to set the multipleOf attribute to `0.001` for our monetary values.

Could we also for the future plan to change the multipleOf attributes from `0.01` to `0.001` ?

Reply:
I definitely recommend multiple of _something_ so that it gets stored as a decimal type.  It is not safe to change the multipleOf in a patch release of the schema, so it is important to think about this carefully for the 1-0-0 schema.

**For Databricks/parquet**: either of these combinations will be stored as a 32 bit decimal value:

```
multipleOf: 0.01
maximum: 9999999

-- or --

multipleOf: 0.001
maximum: 999999
```

You can of course use a larger maximum, and it will get stored as a 64 bit value.  Which is probably OK.

**For BigQuery**: all decimals are stored as 64 bit decimals, so everything from above still applies, but it is less important to limit the maximum.

**For Redshift**: This is disappointing and you won't like it:

- For `multipleOf: 0.01` we store it as a 64 bit decimal value ([link](https://github.com/snowplow/schema-ddl/blob/0.17.1/modules/core/src/main/scala/com.snowplowanalytics/iglu.schemaddl/redshift/generators/TypeSuggestions.scala#L92-L93))
- For multiple of _any other value_ (e.g. of 0.001) then it gets stored as a Double :(

So it seems you need to choose... either multipleOf 0.01 and then you get nice behaviour in all warehouses.  Or multipleOf 0.01, and then you get nice behaviour in most warehouses except for Redshift.

Reply:
I was thinking a bit more... I think we could safely update schema-ddl so it nicely handles `multipleOf: 0.001` for Redshift and BigQuery.  To get the benefit of the change, pipelines will need to be updated to a new version of RDB loader or BQ loader, before starting to send events with this schema.

Let me know if you want us to work on this small change.

Reply:
I have moved the `refund` schema into R141 (PR #1258) to be released after this issue is resolved.